security/webauthn/src/main/java/io/quarkus/ts/security/webauthn/model/WebAuthnCertificate.java

```diff
@@ -0,0 +1,17 @@
+package io.quarkus.ts.security.webauthn.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.ManyToOne;
+
+import io.quarkus.hibernate.reactive.panache.PanacheEntity;
+
+@Entity
+public class WebAuthnCertificate extends PanacheEntity {
+    @ManyToOne
+    public WebAuthnCredential webAuthnCredential;
+
+    /**
+     * The list of X509 certificates encoded as base64url.
+     */
+    public String x5c;
```

Comment:
no idea what `x5c` means, please rename this property

Reply:
Okay, I though about it and it probably means x509certificate. Please rename this property.

Reply:
Renamed it!